eagleproject/core/views.py

```diff
@@ -301,6 +301,28 @@ def make_specific_week_report(request, week):
     return HttpResponse("ok")
 
 
+def remove_specific_month_report(request, month):
+    if not settings.ENABLE_REPORTS:
+        print("Reports disabled on this instance")
+        return HttpResponse("ok")
+
+    year = datetime.datetime.now().year
+    report = AnalyticsReport.objects.get(month=month, year=year)
+    report.delete()
+    return HttpResponse("ok")
+
+
+def remove_specific_week_report(request, week):
+    if not settings.ENABLE_REPORTS:
+        print("Reports disabled on this instance")
+        return HttpResponse("ok")
+
+    year = datetime.datetime.now().year
+    report = AnalyticsReport.objects.get(week=week, year=year)
+    report.delete()
+    return HttpResponse("ok")
```

Comment:
I lack context on these so I'm sorry if I missed something.  Are these reports endpoints operations we want randos to be able to be able to do?

Reply:
they shouldn't be there usually, but it's also possible to generate reports for weeks/months that haven't finished yet which prevents the email being sent, so it's be a temporary measure to get rid of those in time for the email. After that, the whole system should be revamped and they'd be removed. Can also regenerate any report that gets removed

Reply:
> they shouldn't be there usually,

Do you mean the endpoints shouldn't be accessible?  Because I see `ENABLE_REPORTS` set to `true` on prod.

Reply:
I just meant I'd remove them later when changing the way reports/email get generated

Reply:
aight, as long as we recognize the risks and have a plan to fix it.

Reply:
yep, thanks